package.json

```diff
@@ -65,7 +65,8 @@
     "@jsonforms/core": "^3.3.0",
     "@jsonforms/react": "^3.3.0",
     "antd": "^5.14.0",
-    "react": "^17 || ^18"
+    "react": "^17 || ^18",
+    "react-markdown": "^6"
```

Comment:
As long as this version of `react-markdown` has what we need, I think it's fine.

Reply:
![Screenshot 2024-09-10 at 2 48 37 PM](https://github.com/user-attachments/assets/58958e08-d079-4332-ac19-de516208de06)

done!